README.md

```diff
@@ -208,42 +208,42 @@ app.add_flag(option_name,
              help_string="")
 
 app.add_flag(option_name,
-             variable_to_bind_to, // bool, int, 🚧 float, 🚧 vector, 🚧 enum, or 🚧 string-like, or 🚧 anything with a defined conversion from a string
+             variable_to_bind_to, // bool, int, 🆕 float, 🆕 vector, 🆕 enum, or 🆕 string-like, or 🆕 anything with a defined conversion from a string
              help_string="")
 
-app.add_flag_function(option_name, // 🚧
+app.add_flag_function(option_name, // 🆕
              function <void(int64_t count)>,
              help_string="")
 
-app.add_flag_callback(option_name,function<void(void)>,help_string="") // 🚧
+app.add_flag_callback(option_name,function<void(void)>,help_string="") // 🆕
 
 // Add subcommands
 App* subcom = app.add_subcommand(name, description);
 
-Option_group *app.add_option_group(name,description); // 🚧
+Option_group *app.add_option_group(name,description); // 🆕
 
-// 🚧 All add_*set* methods deprecated in CLI11 1.8 - use ->transform(CLI::IsMember) instead
+// 🆕 All add_*set* methods deprecated in CLI11 1.8 - use ->transform(CLI::IsMember) instead
```

Comment:
For clarity sake maybe consider some sort of mark in the readme next to each deprecated function,
maybe ⚠️ or something similar

Reply:
That looks like a good idea. 👍

Reply:
Added, at least to the two places I could think of putting it.